pkg/tunneldriver/driver.go

```diff
@@ -452,6 +452,12 @@ func (td *TunnelDriver) CreateAgentEndpoint(ctx context.Context, name string, sp
 			upstreamProto = string(*spec.Upstream.Protocol)
 		}
 		opts = append(opts, config.WithAppProtocol(upstreamProto))
+
+		// TODO: This should probably be inferred from the scheme in the URL
```

Comment:
Did you mean to leave this comment in? If the URL was already parsed using the helper func, then it should already have a scheme, even if the user used one of the shorthand formats.

Reply:
This was meant to comment on the ngrok-go SDK not respecting the scheme in the URL that we give it to remind me to fix it upstream. Open to removing it or modifying it for clarity.

Reply:
No worries, was just a little confused